### PR TITLE
Shorter descriptions in the schedule

### PR DESCRIPTION
**Pull request by jakearchibald**
I experimented with a auto automated ways to truncate talk descriptions (omg `line-clamp` is so so buggy), then I thought noooope, better to do it manually.

### CODE REVIEW

**Comment by jakearchibald** [comment]
@harleenkbatra08 @vschupmann what do you think? https://deploy-preview-275--cds2019.netlify.com/devsummit/schedule/

File: src/sessions/advancing-the-web-framework-ecosystem.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ speakers:
   - houssein-djirdeh
 start: 2019/11/12 11:10
 end: 2019/11/12 11:45
+description: This talk covers the team's work with Next.js on build and runtime optimizations, what's next in the pipeline, and our plans for other parts of the ecosystem.
```
Comment (surma) [inline src/sessions/advancing-the-web-framework-ecosystem.md]: I’m wondering if they should all have “…” to signify that there is more?

File: src/schedule/script/create-schedule.js [hunk 2]
```diff
@@ -80,37 +80,38 @@ module.exports = function createScheduleHtml(
...
-                          href="${confPath}sessions/${item.fileSlug}"
-                        >
-                          <div class="${classNameMap['sessionItem']}">
-                            <h3 class="${classNameMap['sessionItemTitle']}">
-                              ${item.title}
-                            </h3>
-                            <div>${safe(item.body)}</div>
+                        <div class="${classNameMap.sessionItem}">
+                          <h3 class="${classNameMap.sessionItemTitle}">
+                            <a
```
Comment (surma) [inline src/schedule/script/create-schedule.js]: In the design the session title grey, with this change it is blue.

Comment (jakearchibald) [reply src/schedule/script/create-schedule.js]: Yeah, that was initially unintended, but I think it makes it clearer that it's a link. What do you think @argyleink?

File: src/sessions/advancing-the-web-framework-ecosystem.md (hunk 1 above)
Comment (jakearchibald) [reply src/sessions/advancing-the-web-framework-ecosystem.md]: Yeah. I ballsed that up. Will fix.

**Comment by jakearchibald** [comment]
As a side effect, this will set the meta tags up which will improve social sharing.

File: src/schedule/script/create-schedule.js (hunk 2 above)
Comment (surma) [reply src/schedule/script/create-schedule.js]: Wfm

Comment (argyleink) [reply src/schedule/script/create-schedule.js]: it's very nice, lovely happy accident

File: src/sessions/advancing-the-web-framework-ecosystem.md (hunk 1 above)
Comment (argyleink) [reply src/sessions/advancing-the-web-framework-ecosystem.md]: is ballsed it up a typo or slang i need to adopt?

Comment (jakearchibald) [reply src/sessions/advancing-the-web-framework-ecosystem.md]: https://www.urbandictionary.com/define.php?term=Ballsed%20it%20up 😄

**Pull request merged**